verygoodsecurity/vgs-satellite
# [SAT-185] Remove 'localhost'-restriction for the management API.

## Fixes [SAT-185](https://verygoodsecurity.atlassian.net/browse/SAT-185), #172
## Description of changes in release / Impact of release:
Remove 'localhost'-restriction for the management API.

Test plan:

1. Start VGS Satellite as
```bash
vgs-satellite> docker-compose up --build satellite
```
2. In a separate terminal run (assuming that Stallite container is in `vgs-satellite_default`-network):
```bash
vgs-satellite> docker run --rm -it --network vgs-satellite_default curlimages/curl satellite:8089/flows.json
```
3. Observe `200`-response.

### Is this a breaking change?

### If you answered Yes then describe why is it so
(insert text here)

#### Additional details go here
(insert text here)
